src/main/webapp/index.html

```diff
@@ -16,21 +16,71 @@
       			<h1 id="Main">Welcome to your custom Sequence builder</h1>
 				<div id = "box">
 					<div>
-					  <div id="inputGroup">
-						<select id="Semester">
-							<option selected="true" value="Select an entry Semester" disabled="disabled">Select an entry Semester</option>
-							<option value="Fall">Fall</option>
-							<option value="Winter">Winter</option>
-						</select>
-						<select id="Program">
-							<option selected="true" value="Select a Program" disabled="disabled">Select a Program</option>
-							<option value="Software Engineering">Software Engineering</option>
-							<option value="Computer science">Computer science</option>
-						</select>
-							<div id="checkbox">
-								<input type="checkbox" name="Co-op">Co-op Program?</input>
-							</div>
-					  </div>
+						<div id="inputGroup">
+							<select id="SequenceType">
```

Comment:
we should try to implement this SequenceType select tag to be rendered dynamically based on the sequences we have so that when we update and add more sequences we don't have to keep adding to this list of options every time